### DISCUSSION TITLE
Cannot find how to make it work with react router

### DISCUSSION THREAD

**Discussion by Lunrk** [alex8088/electron-vite#341] [Q&A]
I'm trying to use electron-vite with react-router-dom, in development environment, it works perfectly, but in production, the app first loads the html file and I see the router error page first and the location is a file path ( path to index.html ), then when I click on any link the location (I displayed it on the page for test purposes) change and becomes /pathname, any idea how to deal with that ?

**Comment by alex8088** [comment]
https://electron-vite.org/guide/troubleshooting#vue-router-or-react-router-dom-works-in-development-but-not-production
